src/zentables/zentables.py

```diff
@@ -845,9 +845,9 @@ def _seed_to_rng(seed: Optional[Union[int, Generator]] = None) -> Generator:
         # When there is no seed set, we start a RNG based on system entropy
         return np.random.default_rng()
 
-    if isinstance(seed, int):
+    if isinstance(seed, int) or isinstance(seed, np.int64):
```

Comment:
Oh huh! I had no idea!! But stackoverflow says this is a system issue and you should do `isinstance(seed, np.integer)` : https://stackoverflow.com/questions/37726830/how-to-determine-if-a-number-is-any-type-of-int-core-or-numpy-signed-or-not